core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -0,0 +1,164 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the exploration player, for use in Protractor
+ * tests.
+ */
+
+var forms = require('./forms.js');
+var general = require('./general.js');
+var interactions = require('../../../extensions/interactions/protractor.js');
+
+var ExplorationPlayerPage = function() {
+  var conversationInputAll =
+    element.all(by.css('.protractor-test-conversation-input'));
+  var conversationInput =
```

Comment:
Can we use either this variable or the one above, but not both? They seem to target the same thing.

Reply:
One had element.all and other just element, thats why I used different.  Do both do the same thing?

Reply:
Done

Reply:
They don't do the same thing, but the idea is to find a way to do this that removes the need for having two separate variables.